src/matching_market.sol

```diff
@@ -390,6 +404,18 @@ contract MatchingMarket is MatchingEvents, ExpiringMarket, DSNote {
 
     // ---- Internal Functions ---- //
 
+    function _setMinSell(
+        ERC20 pay_gem,     //token to assign minimum sell amount to
+        uint256 dust
+    )
+        internal
+        returns (bool)
+    {
+        _dust[address(pay_gem)] = dust;
+        emit LogMinSell(address(pay_gem), dust);
+        return true;
+    }
+
```

Comment:
Why not merge this with `function setMinSell(ERC20 pay_gem)`?

Reply:
B/c we need to call it in the constructor.

Reply:
I think the return value could be just removed from here and directly make that the public `setMinSell` to return it.
And honestly I'd remove it totally as I do not see the added value, it will revert or always return `true`.

Reply:
Yes I can get rid of the return value as it doesn't make any sense.